### ISSUE TITLE
Mobile view - index.html

### ISSUE THREAD

**Issue by agnikka** [kukuramba/incode-MrCoffee#2]
1. footer:  social media icons "run away" from the grey background
2.  on some devices letters of two independent lines (of independent blocks of text) merge 
![indeks-footer-ikonki](https://user-images.githubusercontent.com/12581810/122456577-05976880-cfae-11eb-9e6a-548701d1a8e1.png)
![indeks-sklejone-literki](https://user-images.githubusercontent.com/12581810/122456578-062fff00-cfae-11eb-89de-78dd2d596367.png)
together

**Comment by kukuramba** [comment]
Yes, you're right. It's because I didn't check responsiveness on very small devices (screen smaller than 500px).

I've just added new media queries - it's fixed now.

Thanks!
